cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -69,6 +67,10 @@ type NetConf struct {
 
 	// MTU for eth0
 	MTU string `json:"mtu"`
+
+	PluginLogFile string `json:"pluginLogFile"`
+
+	PluginLogLevel string `json:"PluginLogLevel"`
```

Comment:
Please set the json attribute name to `pluginLogLevel` to match the other names.

Reply:
typo, sorry!